dist/api_data.py

```diff
@@ -1099,6 +1099,22 @@ def __ge__(self, other):
     ]),
 ]
 
+wiredtiger_open_live_restore_configuration = [
+    Config('live_restore', '', r'''Live restore configuration options. These options control the
```

Comment:
While I know this is in-progress - in `api_data.py` you might look at `log_configuration_common` and then `connection_reconfigure_log_configuration` and `wiredtiger_open_log_configuration` for things that can or cannot be changed in a `reconfigure` call (like the number of threads).

Reply:
I did take a look around and have seen the distinction between things that could be configured at run-time vs only on open. At this stage I don't think we need to support tuning the live restore threads at runtime so I'd prefer to only implement that if the need arises.

Reply:
That seems like a good place to start and you're right it may not be needed, because the live restore is a "temporary" condition. Make a comment of that reason someplace.

Reply:
Added comment: 7373b0944